packages/contracts-ops/contracts/test/RebootEthereumBridgeRouter.t.sol

```diff
@@ -19,31 +19,35 @@ contract EthereumBridgeRouterRebootTest is RebootTest, BridgeRouterTest {
     address bridgeRouterHarnessImpl;
     address tokenRegistryHarnessImpl;
 
-    string constant ethereum = "ethereum";
-
     function setUp() public override(BridgeRouterBaseTest, NomadTest) {
         setUpReboot("ethBridgeRouter");
+        require(
+            keccak256(bytes(localDomainName)) == keccak256(bytes("ethereum")),
+            "not ethereum bridge router"
```

Comment:
i don't think there's a `skipTest` cheat code, so allowing it to fail at setup (thus not run any tests) with a clear error message

Reply:
```
modifier skipNotEthereum() {
    if (keccak256(bytes(localDomainName)) == keccak256(bytes("ethereum"))) {
        _;
    }
}
```

Reply:
we don't have access to `localDomainName` in the inherited unit tests. also, that would mean that the unit tests don't run